diff --git a/index.d.ts b/index.d.ts
index b3ab7e3d5bd85285ffe3a5bbdc30a603680d57d8..04d50e1a5f145e699648eedf47c37197f65d6f1f 100644
--- a/index.d.ts
+++ b/index.d.ts
@@ -982,13 +982,13 @@ export interface MatchPattern {
 export interface Span {
     start: number;
     end: number;
-    ctxt: number;
 }
 export interface Node {
     type: string;
 }
 export interface HasSpan {
     span: Span;
+    ctxt?: number
 }
 export interface HasDecorator {
     decorators?: Decorator[];
